gradle-plugin/kubernetes/src/main/java/org/eclipse/jkube/gradle/plugin/KubernetesExtension.java

```diff
@@ -74,6 +77,40 @@
 
   public abstract Property<Boolean> getUseColor();
 
+  public abstract Property<Integer> getMaxConnections();
+
+  public abstract Property<String> getFilter();
+
+  public abstract Property<String> getApiVersion();
+
+  public abstract Property<JKubeBuildStrategy> getBuildStrategy();
+
+  public abstract Property<String> getBuildRecreate();
+
+  public abstract Property<String> getImagePullPolicy();
+
+  public abstract Property<String> getAutoPull();
+
+  public abstract Property<String> getDockerHost();
+
+  public abstract Property<String> getCertPath();
+
+  public abstract Property<String> getMinimalApiVersion();
+
+  public abstract Property<Boolean> getSkipMachine();
+
+  public abstract Property<Boolean> getForcePull();
+
+  public abstract Property<Boolean> getSkipExtendedAuth();
+
+  public abstract Property<String> getPullRegistry();
+
+  public abstract Property<String> getSourceDirectory();
+
+  public abstract Property<String> getOutputDirectory();
```

Comment:
I know these properties are named the same in the Maven Mojos, but the names are confusing.

I suggest to rename them to something like:
```suggestion
  public abstract Property<String> getBuildSourceDirectory();

  public abstract Property<String> getBuildOutputDirectory();
```

Without this nuance they can be confused with the project source and build directory.